### PR TITLE
Updated information for CoIoT (Shelly Motion)

### PR DESCRIPTION
**Pull request by tubalainen**
Configuration information.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by tubalainen** [comment]
Ping @thecode 
ref: https://github.com/home-assistant/core/issues/47271

File: source/_integrations/shelly.markdown [hunk 1]
```diff
@@ -113,6 +113,12 @@ Not all devices support all input events. You can check on [Shelly API Reference
 
 </div>
 
+## Action driven battery powered devices (CoIoT) (initial support)
+For action driven Shelly battery powered devices such as the Shelly Motion it is required that the "CoIoT" option is set so that the sensor can send action events, such as "motion detection" to Home Assistant.
```
Comment (thecode) [inline source/_integrations/shelly.markdown]: The "CoIoT" is required for the sensor to work, not only for actions. It is also need to be set before adding the device to Home Assistant, I think it worth mentioning this.
I am not sure how to correctly call this new category, for now it is only Shelly Motion, but it is planned to add this support for other devices also.
Maybe change the header:
## Silabs based powered devices (CoIoT) (initial support)

**Comment by thecode** [comment]
I think it is better to include a small picture, something like this:
image

**Review by tubalainen — COMMENTED** [review]
Ok, updated and clarified! Thanks

File: source/_integrations/shelly.markdown [hunk 2]
```diff
@@ -113,6 +113,11 @@ Not all devices support all input events. You can check on [Shelly API Reference
 
 </div>
 
+## Silabs based powered devices (CoIoT) (initial support)
```
Comment (chemelli74) [inline source/_integrations/shelly.markdown]: Why `(initial support)` ? Sounds like is incomplete, while it's not

**Review by tubalainen — COMMENTED** [review]
Removed the "initial support" according to @chemelli74 feedback.

File: source/_integrations/shelly.markdown [hunk 3]
```diff
@@ -113,6 +113,11 @@ Not all devices support all input events. You can check on [Shelly API Reference
 
 </div>
 
+## Silabs based powered devices (CoIoT)
+For Silabs based devices such as the Shelly Motion it is required to enable "CoIoT" in the device for it to work with Home Assistant.
+To setup "CoIoT" on your Shelly device: Navigate to the local IP-address of your Shelly device, "Internet & Security", "COIOT", check the box, enter your local Home Assistant IP-Address in the "remote address" field and push "save".
```
Comment (thecode) [inline source/_integrations/shelly.markdown]:
```suggestion
To setup "CoIoT" on your Shelly device: Navigate to the local IP-address of your Shelly device, "Internet & Security", "COIOT", check the box, enter your local Home Assistant IP-Address in the "remote address" field and push "save".

```
Need a blank line else the image and the text are mixed.
image

**Review by thecode — DISMISSED** [review]
LGTM
Thanks!

**Review by chemelli74 — DISMISSED** [review]
Good job! Thx !

File: source/_integrations/shelly.markdown [hunk 4]
```diff
@@ -113,6 +113,12 @@ Not all devices support all input events. You can check on [Shelly API Reference
 
 </div>
 
+## Silabs based powered devices (CoIoT)
+For Silabs based devices such as the Shelly Motion it is required to enable "CoIoT" in the device for it to work with Home Assistant.
```
Comment (bieniu) [inline source/_integrations/shelly.markdown]:
```suggestion
For Silabs based devices such as the Shelly Motion it is required to enable **CoIoT** in the device for it to work with Home Assistant.
```

File: source/_integrations/shelly.markdown [hunk 5]
```diff
@@ -113,6 +113,12 @@ Not all devices support all input events. You can check on [Shelly API Reference
...
+To setup "CoIoT" on your Shelly device: Navigate to the local IP-address of your Shelly device, "Internet & Security", "COIOT", check the box, enter your local Home Assistant IP-Address in the "remote address" field and push "save".
```
Comment (bieniu) [inline source/_integrations/shelly.markdown]:
```suggestion
To setup **CoIoT** on your Shelly device: navigate to the local IP-address of your Shelly device, **Internet & Security** >> **COIOT**, check the box, enter your local Home Assistant IP-Address in the **Remote address** field and push **Save**.
```

Comment (tubalainen) [reply source/_integrations/shelly.markdown]: Looks good.

**Review by tubalainen — COMMENTED** [review]
LGTM

**Review by thecode — APPROVED** [review]

**Comment by bieniu** [comment]
It seems to me that an `IP address` (not` IP-address`) is used in the documentation of other integrations. I'm not sure it matters.

**Comment by bieniu** [comment]
This PR is still not merged and in the meantime Shelly firmware 1.10.0 has been released and information about CoIoT/unicast applies to all Shelly devices. I have prepared a [PR](https://github.com/home-assistant/home-assistant.io/pull/17083) that adds this information comprehensively.

**Comment by frenck** [comment]
Superseded by #17083

**Pull request closed without merging**